website/content/api-docs/secret/pki.mdx

```diff
@@ -3204,6 +3208,66 @@ $ curl \
     http://127.0.0.1:8200/v1/pki/tidy
 ```
 
+### Configure Automatic Tidy
```

Comment:
Should we add a small blurb that the existing cancellation of tidy operations also work with automatically started tidy ops?

Reply:
Tidy cancellation is not yet implemented :-)